### PR TITLE
Added homework_13_060123

### PR DESCRIPTION
**Pull request by kate-lvlup**
Hi Dima,
Could you check my homework13?

Thanks in advance!

### CODE REVIEW

File: homework_13_060123/src/Data.java [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+import java.util.Scanner;
+
+public class Data {
+
+    public static void readData() {
+        Scanner scanner = new Scanner(System.in);
+        int[] array = new int[10];
+        System.out.println("Enter the numbers:");
+        int i = 0;
+        int j;
```
Comment (YeaaaaH) [inline homework_13_060123/src/Data.java:25]: you can declare i and j inside loop

File: homework_13_060123/src/TestException.java [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+            int num = Integer.parseInt("XYZ");
+            System.out.println(num);
+        } catch (NumberFormatException e) {
+            e.printStackTrace();
+        }
+    }
+
+    public static void divideByZero() {
+        int res = 5 / 0;
+        System.out.println(res);
```
Comment (YeaaaaH) [inline homework_13_060123/src/TestException.java:65]: wrap with try catch inside the method, not in main

**Review by YeaaaaH — CHANGES_REQUESTED** [review]
small fixes needed

File: homework_13_060123/src/TestException.java (hunk 2 above)
Comment (kate-lvlup) [reply homework_13_060123/src/TestException.java:65]: fixed

**Review by YeaaaaH — APPROVED** [review]

**Pull request merged**